core/components/seosuite/bootstrap.php

```diff
@@ -1,8 +1,11 @@
 <?php
+/**
+ * @see \MODX\Revolution\modX::_initNamespaces()
+ */
 require_once __DIR__ . '/vendor/autoload.php';
 
 // Add your classes to modx's autoloader
-$modx->addPackage('Sterc\SeoSuite\Model', $namespace['path'] . 'src/', null, 'Sterc\\SeoSuite\\');
+$modx->addPackage('Sterc\SeoSuite\Model', __DIR__ . '/src/', null, 'Sterc\\SeoSuite\\');
```

Comment:
Not really a required change, but felt like there was no real need to depend on the array value while we could use relative path